plasmapy/physics/parameters.py

```diff
@@ -1521,8 +1551,10 @@ def lower_hybrid_frequency(B, n_i, ion='p+'):
         incompatible dimensions, or ion cannot be used to identify an
         ion or isotope.
 
-    UserWarning
-        If units are not provided and SI units are assumed.
+    Warns
+    -----
+    ~astropy.units.UnitsWarning
+        If units are not provided and SI units are assumed
```

Comment:
I give up, just ctrl+F it :stuck_out_tongue_closed_eyes:

Reply:
Haha, sorry!